pnpm-lock.yaml

```diff
@@ -360,10 +360,65 @@ importers:
         version: 5.2.2
 
   fixtures/webstudio-remix-vercel:
+    dependencies:
+      '@remix-run/css-bundle':
```

Comment:
seems like @remix-run/css-bundle and @remix-run/eslint-config are redundant here